js/table/tableview.ts

```diff
@@ -2482,6 +2489,23 @@ export class TableView {
         }
     }
 
+    /**
+     * Get current user money amount
+     */
+    private async getAccountMoney() {
+        const self = this;
```

Comment:
Using of `self` is no longer needed. Since we use async/await and `this` in this case correctly preserved. Also this variable is never used.